test/smoke/main_test.go

```diff
@@ -15,12 +15,64 @@
 package operator
 
 import (
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
 	"github.com/cucumber/godog"
+	"github.com/cucumber/godog/colors"
 
 	"github.com/kiegroup/kogito-cloud-operator/test/smoke/framework"
 	"github.com/kiegroup/kogito-cloud-operator/test/smoke/steps"
 )
 
+const (
+	defaultDisabledTag = "~@disabled"
+)
+
+var opt = godog.Options{
+	Output:    colors.Colored(os.Stdout),
+	Format:    "progress",
+	Randomize: time.Now().UTC().UnixNano(),
+	Tags:      defaultDisabledTag,
+}
+
+func init() {
+	godog.BindFlags("godog.", flag.CommandLine, &opt)
+	framework.BindEnvFlags(flag.CommandLine)
+}
+
+func TestMain(m *testing.M) {
+	flag.Parse()
+	opt.Paths = flag.Args()
+
+	if !strings.Contains(opt.Tags, defaultDisabledTag) {
+		if opt.Tags != "" {
+			opt.Tags += " && "
+		}
+		opt.Tags += defaultDisabledTag
+	}
+
+	// Check CLI binary is existing
+	if exits, err := framework.CheckCliBinaryExist(); err != nil {
```

Comment:
This means that CLI binary is required for running any test, right?

Reply:
yes... will modify that...